README.md

```diff
@@ -1,23 +1,24 @@
-# BROADEN.IO Backend
+[![eslint: airbnb](https://img.shields.io/badge/eslint-airbnb-%23fd5c63.svg)](https://www.npmjs.com/package/eslint-config-airbnb)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-broaden.io
 
-## Table of Contents
+# Backend
```

Comment:
I think it is necessary to have Broaden.io Backend as the title because Backend is very generic.